tensorrt_llm/_torch/disaggregation/native/py_cache_transceiver.py

```
@@ -0,0 +1,298 @@
+            to_complete_request_ids = list(self.recv_sessions.keys())
+        completed_request_ids = []
+        failed_request_ids = []
+        for request_id in to_complete_request_ids:
+            future = self.recv_sessions[request_id]._kv_tasks[self.recv_task_ids[request_id]].future
+            try:
+                sync_status = future.result()
+                if sync_status == "SUCCESS":
+                    completed_request_ids.append(request_id)
+                else:
+                    failed_request_ids.append(request_id)
+            except Exception:
```

Ruff (0.14.14)

[warning] Do not catch blind exception: `Exception`

(BLE001)